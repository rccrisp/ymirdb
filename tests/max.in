set animals567 5 1 10 -100 -101 37 2 4
max animals567
pluck animals567 6
max animals567
bye