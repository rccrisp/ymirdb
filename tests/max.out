> ok

> 37

> 37

> 10

> bye
